index.d.ts

```diff
@@ -0,0 +1,44 @@
+declare module '@ottofeller/epubjs-rn' {
+  import React from 'react';
+
+  interface Props {
+    style?: object;
+    src?: string;
+    flow?: 'paginated' | 'scrolled';
+    location?: string;
+    onLocationChange?(): void;
+    onLocationsReady?(): void;
+    onViewAdded?(): void;
+    beforeViewRemoved?(): void;
+    width: number;
+    height: number;
+    onReady?(): void;
+    themes?: object;
+    theme?: string;
+    fontSize?: number;
+    font?: string;
+    stylesheet?: string;
+    script?: string;
+    minSpreadWidth?: number;
+    gap?: number;
+    onPress?(ev: object): void;
```

Comment:
onPress must receive other props and not a event object. See the file: https://github.com/ottofeller/epubjs-rn/blob/master/src/Rendition.js#L346

```js
case "press": {
        this.props.onPress && this.props.onPress(decoded.cfi, decoded.position, this);
        break;
}
```

The correct props to ts definition file must be:
```ts
onPress?(cfi: string, position {x: number; y: number;}, rendition: Rendition): void;
```

But, now, we have a potential problem: in this definition, we not have a exported Rendition (as interface or as Component).

Reply:
And, complementing, the Epub component onPress event is parsed directly to the Rendition component, then, to see better the props of the Epub, is needed to see the behaviour of the Rendition.

Reply:
The same occur to other events in those TypeScript definitions.